templates/admin/html-variation-price.php

```diff
@@ -15,50 +15,33 @@
-
-		<label for="variable_subscription_trial_period[<?php echo esc_attr( $loop ); ?>]" class="wcs_hidden_label"><?php esc_html_e( 'Subscription trial period:', 'woocommerce-subscriptions' ); ?></label>
-		<select name="variable_subscription_trial_period[<?php echo esc_attr( $loop ); ?>]" class="wc_input_subscription_trial_period wc-enhanced-select">
-		<?php foreach ( wcs_get_available_time_periods() as $key => $value ) : ?>
-			<option value="<?php echo esc_attr( $key ); ?>" <?php selected( $key, WC_Subscriptions_Product::get_trial_period( $variation_product ) ); ?>><?php echo esc_html( $value ); ?></option>
-		<?php endforeach; ?>
-		</select>
-	</p>
-</div>
 <div class="variable_subscription_pricing variable_subscription_pricing_2_3 show_if_variable-subscription" style="display: none">
-	<p class="form-row form-row-first form-field show_if_variable-subscription _subscription_price_field">
+	<p class="form-row multiple_fields form-row-first form-field show_if_variable-subscription _subscription_price_field">
 		<label for="variable_subscription_price[<?php echo esc_attr( $loop ); ?>]">
 			<?php
 			// translators: placeholder is a currency symbol / code
 			printf( esc_html__( 'Subscription price (%s)', 'woocommerce-subscriptions' ), esc_html( get_woocommerce_currency_symbol() ) );
 			?>
+			<?php echo wcs_help_tip( _x( 'Choose the subscription price, billing interval and period.', 'woocommerce-subscriptions' ) ); ?>
 		</label>
-		<input type="text" class="wc_input_price wc_input_subscription_price" name="variable_subscription_price[<?php echo esc_attr( $loop ); ?>]" value="<?php echo esc_attr( wc_format_localized_price( WC_Subscriptions_Product::get_regular_price( $variation_product ) ) ); ?>" placeholder="<?php echo esc_attr_x( 'e.g. 9.90', 'example price', 'woocommerce-subscriptions' ); ?>">
 
-		<label for="variable_subscription_period_interval[<?php echo esc_attr( $loop ); ?>]" class="wcs_hidden_label"><?php esc_html_e( 'Billing interval:', 'woocommerce-subscriptions' ); ?></label>
-		<select name="variable_subscription_period_interval[<?php echo esc_attr( $loop ); ?>]" class="wc_input_subscription_period_interval wc-enhanced-select">
-		<?php foreach ( wcs_get_subscription_period_interval_strings() as $key => $value ) : ?>
-			<option value="<?php echo esc_attr( $key ); ?>" <?php selected( $key, WC_Subscriptions_Product::get_interval( $variation_product ) ); ?>><?php echo esc_html( $value ); ?></option>
-		<?php endforeach; ?>
-		</select>
+		<span class="wrap">
+			<input type="text" class="wc_input_price wc_input_subscription_price" name="variable_subscription_price[<?php echo esc_attr( $loop ); ?>]" value="<?php echo esc_attr( wc_format_localized_price( WC_Subscriptions_Product::get_regular_price( $variation_product ) ) ); ?>" placeholder="<?php echo esc_attr_x( 'e.g.', 'example price', 'woocommerce-subscriptions' ); ?> <?php echo esc_attr( wc_format_localized_price( '9.90' ) ); ?>">
```

Comment:
```suggestion
			<?php // Translators: %s: formatted example price value. ?>
			<input type="text" class="wc_input_price wc_input_subscription_price" name="variable_subscription_price[<?php echo esc_attr( $loop ); ?>]" value="<?php echo esc_attr( wc_format_localized_price( WC_Subscriptions_Product::get_regular_price( $variation_product ) ) ); ?>" placeholder="<?php printf( esc_attr_x( 'e.g. %s', 'example price', 'woocommerce-subscriptions' ), wc_format_localized_price( '9.90' ) ); ?>">
```

Same commit suggestion here - using the same format `printf( esc_attr_x( 'e.g. %s', ... ) )`